GOVERNANCE.md

```diff
@@ -211,11 +212,19 @@ The reviewers will determine responses based on the information gathered and the
 
 ## OpenCollective fund allocation
 
-- Funds will be allocated for project-specific services such as domain registration and website hosting.
-- Core contributors are allowed to claim expenses related to conferences where they talk about Biome, or Biome workshops that they run. Expenses that can be claimed are but not limited to: trip (plane, train, etc.), meals, transport, parking.
+**Funds will be allocated for project-specific services**:
+  - Domain registrations
+  - Website hosting
+  - Password manager to manage secrets and passwords among Lead members
+
+Core contributors are allowed to claim expenses related to conferences where they talk about Biome, or Biome workshops that they run. Expenses that can be claimed are but not limited to: trip (plane, train, etc.), meals, transport, parking.
   - The person will have to provide proof of attendance (badge, video, etc.);
   - Expenses will have to be uploaded via the open collective website.
-- Expenses will be approved by the [leads](#lead).
+  - Only the expenses that afforded in the days of the workspace/conference are accepted:
+    - If a conference lasts multiple days and the member decides to attend *both* days, and the member speaks only the first day, only the expenses of the first day should be taken in consideration. In this example, if the member booked three nights in a hotel to attend these two days, the expense should be divided by 3  - three nights - and only expense 2 days: night before the conference, plus the night of the day the member speaks.
+    - If a conference lasts one day and the member decides to make their performance longer, all the expenses afforded in the other days won't be accepted. Only 2 nights of accommodation are accepted.
+
+Expenses must be approved by at least one [leads](#lead). In the presence of one single lead, the lead will ask one core contributor to fact-check the honesty of expenses, and leave a comment in the Open Collective expense page. In case of multiple leads, the lead submitting the expense must seek the approval of at least one other lead.
```

Comment:
```suggestion
Expenses must be approved by at least one [lead](#lead). In the presence of one single lead, the lead will ask one core contributor to fact-check the honesty of expenses, and leave a comment in the Open Collective expense page. In case of multiple leads, the lead submitting the expense must seek the approval of at least one other lead.
```